PantryServer/Models/Cart.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+
+//This will allow users to place albums in their cart without registering, but they’ll need to register to complete checkout
```

Comment:
if your going to copy paste code - at least change the references... we don't have albums in pantry - yet :)